packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <div v-if="!showEmptyState" class="flex flex-col gap-4">
+      <WorkspaceHeader
+        v-if="workspace"
+        :icon="Squares2X2Icon"
+        :workspace-info="workspace"
+        :project-count="projects?.totalCount"
+      />
+      <div class="flex flex-col sm:flex-row gap-2 sm:items-center justify-between">
+        <div class="flex flex-col sm:flex-row gap-2">
+          <FormTextInput
+            name="modelsearch"
+            :show-label="false"
+            placeholder="Search..."
+            :custom-icon="MagnifyingGlassIcon"
+            color="foundation"
+            wrapper-classes="grow md:grow-0 md:w-60"
+            :show-clear="!!search"
+            :model-value="bind.modelValue"
+            v-on="on"
+          ></FormTextInput>
+          <!-- <FormSelectProjectRoles
+            v-if="!showEmptyState"
+            v-model="selectedRoles"
+            class="md:w-56 grow md:grow-0"
+            fixed-height
+          /> -->
+        </div>
+        <FormButton v-if="!isGuest" @click="openNewProject = true">
+          New project
+        </FormButton>
+      </div>
+    </div>
+
+    <CommonLoadingBar :loading="showLoadingBar" class="my-2" />
```

Comment:
There's still a large amount of duplication between this and ProjectsDashboard, but in an effort to get this out quicker lets leave it as is for now